ui/src/scss/_color_variables.scss

```diff
@@ -182,3 +182,5 @@ $icon-btn-color: #42526e;
 $active-brand-color: #26A775;
 $light-red-color: #FCE3E3;
 $error-note: #ff0000;
+$theme-light-secondary-text-color:#A0ABBB;
```

Comment:
color names should be equivalent to it's nature instead of where it's used. so, same code can be used in multiple places

Reply:
the name should be relevant to the color? @guganabu

Reply:
yes, like the variable should be equivalent to color code but not context of where it's been used

Reply:
can you plz resolve it @nixonakash01 ?